code/processes/housekeeping.q

```diff
@@ -60,6 +73,18 @@ wrapper:{[DICT]
 
 //FUNCTIONS FOR LINUX
 
+
+
+//-compress by calling .cmp.compress function defined
+kdbzip:{[FILE]   
+	 @[{.lg.o[`housekeeping;"compressing ",x]; .cmp.compress[hsym `$x ;2;17;4;hcount hsym `$x]};
```

Comment:
```hsym`$x``` is used twice, could save as a temp var and call that + can remove some whitespace.